stix_shifter_utils/stix_translation/src/utils/transformers.py

```diff
@@ -302,6 +302,29 @@ def transform(obj):
             return result
         return obj
 
+
+class CheckIPv6(ValueTransformer):
+    """A value transformer for filtering-out from a list all values which are not valid IPv6 values"""
```

Comment:
How is this class any different from `FilterIPv6List`? They both have the same description comment. Are you just trying to see if the value is indeed an IPv6?

Reply:
Will change the description. This does the same as `FilterIPv6List` but for one element. If the value is not IPv6, the value will be transformed to null.